doc/api/errors.md

```diff
@@ -1831,8 +1831,13 @@ unknown file extension.
 
 Creation of a [`zlib`][] object failed due to incorrect configuration.
 
-[`'uncaughtException'`]: process.html#process_event_uncaughtexception
+### MODULE_NOT_FOUND
```

Comment:
It is very unfortunate that this does not have the `ERR_` prefix :-(

Reply:
The code is 7 years old so we can't blame anyone for that : https://github.com/nodejs/node/commit/3f987cd925dd342b4695afd9c221bc19e76bdf7b